zeebe/util/src/main/java/io/camunda/zeebe/util/micrometer/MicrometerUtil.java

```diff
@@ -96,6 +98,25 @@ public static void closeRegistry(final MeterRegistry registry) {
     registry.close();
   }
 
+  public static Duration[] exponentialBucketDuration(
```

Comment:
:x: Please document what it returns, and esp. why/when you should use it.

Reply:
I'll add it

Reply:
:heavy_check_mark:  added